web-app/src/assets/base.css

```diff
@@ -79,7 +57,7 @@ body {
     'Droid Sans',
     'Helvetica Neue',
     sans-serif;
-  font-size: 15px;
+  font-size: 16px;
```

Comment:
I think I'll set this to 14px - I'll work on that sooner rather than later otherwise it will be a lot of styles updates